### PR TITLE
fix: do noted enhancements

### PR DESCRIPTION
**Pull request by sonle-geekyants**
**- What I did**
- Do the noted enhancements in [#839](https://github.com/atsign-foundation/apps/issues/839) and resolve remaining issues in [#835](https://github.com/atsign-foundation/apps/issues/835)

**- How I did it**

**- How to verify it**

**- Description for the changelog**

### CODE REVIEW

File: lib/utils/file_utils.dart [hunk 1]
```diff
@@ -25,31 +25,17 @@ class FileUtils {
...
-      context: NavService.navKey.currentContext!,
-      backgroundColor: Colors.white,
-      builder: (context) => EditBottomSheet(
-        onConfirmation: () async {
-          var file = File(filePath);
-          if (await file.exists()) {
-            await file.delete();
-          }
-          if (fileTransferId != null) {
-            await Provider.of<MyFilesProvider>(
```
Comment (Sardar1208) [inline lib/utils/file_utils.dart:38]: why did we remove this ? 
This function deletes the file from the local provider

File: lib/utils/file_utils.dart [hunk 2]
```diff
@@ -25,31 +25,17 @@ class FileUtils {
...
-                    listen: false)
-                .getAllFiles();
-          }
-          onComplete?.call();
-        },
-        deleteMessage: TextStrings.deleteFileConfirmationMsgMyFiles,
-      ),
+    await CommonUtilityFunctions().showConfirmationDialog(
+      () {
+        File(filePath).deleteSync();
```
Comment (Sardar1208) [inline lib/utils/file_utils.dart]: Can you check for file.exists() before we call delete?

File: lib/utils/file_utils.dart (hunk 1 above)
Comment (sonle-geekyants) [reply lib/utils/file_utils.dart:38]: @Sardar1208 When I do testing, there are no problems have shown up yet. I will add them back for sure

Comment (Sardar1208) [reply lib/utils/file_utils.dart:38]: Okay sure

**Review by Sardar1208 — APPROVED** [review]

**Pull request merged**